src/components/AnonymousChat/child/ChatItemTargetText.tsx

```diff
@@ -14,6 +14,7 @@ import {
 } from './ChatItemAttachmentStyles';
 import ChatItemAttachment from './ChatItemAttachment';
 import dimen from '../../../utils/dimen';
+import {LinkableText} from '../../LinkableText';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `LinkableText` is correct and it's a good practice to use absolute imports instead of relative ones. However, it seems like the path is relative. If your project supports absolute imports, consider using them for better readability and maintainability.
```diff
- import {LinkableText} from '../../LinkableText';
+ import {LinkableText} from 'src/components/LinkableText';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->